docs/REST API/rest-api.md

```diff
@@ -7015,6 +7077,53 @@ Deletes the session specified by ID. When the user associated with the session n
   "count": 43
 }
 ```
+
+### Get software title
```

Comment:
This reminded me, we need an API endpoint to return the count and timestamp for the Software titles table:

<img width="168" alt="Screenshot 2023-11-14 at 8 27 58 PM" src="https://github.com/fleetdm/fleet/assets/47070608/4d6a2f22-c04b-4716-b48e-e23c0f18f818">

Looks like we use the [`GET /software/count` API endpoint](https://fleetdm.com/docs/rest-api/rest-api#count-software) for the **Software** page today. 

I think we add a `count` value to the `GET /software/titles` and `GET /software/versions` endpoints so the user doesn't have to call a separate endpoint. Then, we can deprecate `GET /software/count`

What do you think?

Reply:
We already have `counts_updated_at` in the `GET /software/`. Makes sense to me to have `count` as well.

Reply:
> We already have `counts_updated_at` in the GET /software/

Oh. Then what do we use `GET /software/count` for today? Can we get rid of it?

Reply:
@noahtalerman That would be breaking change. Should we leave this one and display only new `GET /software/titles` and `GET /software/versions`endpoints, as we do for `GET /software`?

Reply:
Yeah that makes sense to me. 

We can remove `GET /software/count` from the docs.